src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,168 @@
+
+`example-component/orion-fusion-theme/ExampleComponent.module.scss`
+
+```scss
+:local {
+  .orion-fusion-theme {
+    --terra-application-example-component-background-color: #f6f8fa;
+    --terra-application-example-component-color: rgb(36, 41, 46);
+    --terra-application-example-component-font-size: 2rem;
+  }
+}
+```
+
+`example-component/clinical-lowlight-theme/ExampleComponent.module.scss`
+
+```scss
+:local {
+  .clinical-lowlight-theme {
+    --terra-application-example-component-background-color: rgb(34, 42, 46);
+    --terra-application-example-component-color: rgb(178, 181, 182);
+    --terra-application-example-component-font-size: 1rem;
+  }
+}
+```
+
+Include the new theme files in the original SCSS file.
+
+`example-component/ExampleComponent.module.scss`
+
+```scss
+// Import each theme.
+@import './clinical-lowlight-theme/ExampleComponent.module';
+@import './orion-fusion-theme/ExampleComponent.module';
+
+:local {
+  .example-component {
+    background-color: var(--terra-application-example-component-background-color, #fff);
+    color: var(--terra-application-example-component-color, inherit);
+    font-size: var(--terra-application-example-component-font-size, 1rem);
+  }
+}
+```
+
+The recommended file structure ends up looking like this:
+
+```
+.
+├── example-component
+│   ├── ExampleComponent.jsx
+│   ├── ExampleComponent.module.scss
+│   ├── clinical-lowlight-theme
+│   │   └── ExampleComponent.module.scss
+│   └── orion-fusion-theme
+│       └── ExampleComponent.module.scss
+```
+
+## Webpack
+
+To enable themes, teams must define a `terra-theme.config.js` configuration file. This file is read by the [terra-toolkit postcss loader](https://github.com/cerner/terra-toolkit/tree/master/config/webpack/postcss) to aggregate themes. This file must be defined for theme CSS assets to be included in the webpack bundle. The loader runs automatically for webpack configurations using [terra-toolkit's webpack config](https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js).
+
+```js
+const themeConfig = {
+  theme: 'terra-dark-theme', // The default theme to be enabled on page load.
+  scoped: ['terra-light-theme', 'terra-lowlight-theme'], // An array of scoped themes. Note: Scoped themes do not work in IE 10.
+};
+
+module.exports = themeConfig;
+```
+
+## Example
+
+Toggle the theme from the dropdown at the top of the page to see changes. The full example code can be seen [here](https://github.com/cerner/terra-application/tree/master/src/terra-dev-site/doc/application/how-to.2/example).
+
+<ExampleComponent />
```

Comment:
you can set this to be open by default.

```suggestion
Toggle the theme from the dropdown at the top of the page to see changes. The full example code can be seen [here](https://github.com/cerner/terra-application/tree/master/src/terra-dev-site/doc/application/how-to.2/example).

<ExampleComponent isExpanded />
```

Reply:
I went back and forth on having it open by default. Seems like it might draw more attention to have open initially, updated:

https://github.com/cerner/terra-application/pull/25/commits/bbbd86df8667fc13a0b7c0ee3ca9997e9365209f